src/commands/up.rs

```diff
@@ -630,11 +654,8 @@ fn trigger_command(trigger_type: &str) -> Vec<String> {
     vec!["trigger".to_owned(), trigger_type.to_owned()]
 }
 
-fn trigger_command_for_resolved_app_source(
-    resolved: &ResolvedAppSource,
-) -> Result<Vec<Vec<String>>> {
-    let trigger_type = resolved.trigger_types()?;
-    trigger_type
+fn trigger_command_for_resolved_app_source(trigger_types: Vec<&str>) -> Result<Vec<Vec<String>>> {
```

Comment:
Is the name of this function still correct?

Reply:
Updated